dotcom-rendering/src/components/LoopVideoPlayer.tsx

```diff
@@ -228,61 +238,65 @@ export const LoopVideoPlayer = forwardRef(
 					)}
 					{FallbackImageComponent}
 				</video>
-				{!!activeCue?.text && (
+				{showSubtitles && (
 					<SubtitleOverlay
 						text={activeCue.text}
 						subtitleSize={subtitleSize}
 					/>
 				)}
-				{ref && 'current' in ref && ref.current && isPlayable && (
-					<>
-						{/* Play icon */}
-						{showPlayIcon && (
-							<button
-								type="button"
-								onClick={handlePlayPauseClick}
-								css={playIconStyles}
-								data-link-name={`gu-video-loop-play-${atomId}`}
-								data-testid="play-icon"
-							>
-								<PlayIcon iconWidth="narrow" />
-							</button>
-						)}
-						{/* Progress bar */}
-						<LoopVideoProgressBar
-							videoId={loopVideoId}
-							currentTime={currentTime}
-							duration={ref.current.duration}
-						/>
-						{/* Audio icon */}
-						{AudioIcon && (
-							<button
-								type="button"
-								onClick={handleAudioClick}
-								css={audioButtonStyles}
-								data-link-name={`gu-video-loop-${
-									isMuted ? 'unmute' : 'mute'
-								}-${atomId}`}
-							>
-								<div
-									css={audioIconContainerStyles}
-									data-testid={`${
+				{ref &&
+					'current' in ref &&
+					ref.current &&
+					isPlayable &&
+					!isCinemagraph && (
```

Comment:
Do we need to check isCinemagraph here given we set this at the wrapper above 
			`showPlayIcon={!isCinemagraph ? showPlayIcon : false}`